app/controllers/candidate_interface/referees_controller.rb

```diff
@@ -109,6 +114,20 @@ def review
       @application_form = current_candidate.current_application
     end
 
+    def complete
+      if current_application.application_references.count >= ApplicationForm::MINIMUM_COMPLETE_REFERENCES
+        current_application.update!(application_form_params)
+
+        redirect_to candidate_interface_application_form_path
+      else
+        flash[:warning] = 'You can’t mark this section complete without adding two referees.'
```

Comment:
Could we add a tiny bit of gold plating to this and make the `two referees` based on `ApplicationForm::MINIMUM_COMPLETE_REFERENCES`?